Content.Server/_DV/CartridgeLoader/Cartridges/NanoChatCartridgeSystem.cs

```diff
@@ -506,14 +599,12 @@ private void UpdateUIForAllCards()
     }
 
     /// <summary>
-    ///     Truncates a message to the notification maximum length.
+    ///     Truncates a string to a maximum length.
     /// </summary>
-    private static string TruncateMessage(string message)
-    {
-        return message.Length <= NotificationMaxLength
-            ? message
-            : message[..(NotificationMaxLength - 4)] + " [...]";
-    }
+    private static string Truncate(string text, int maxLength, string overflowText = "...") =>
+        text.Length <= maxLength
+            ? text
+            : text[..(maxLength - overflowText.Length)] + overflowText;
```

Comment:
put this in shared so ui doesnt copy paste it

Reply:
Shared as in SharedNanoChatSystem?

Reply:
yes